frontend/public/components/cloud-services/catalog-source.tsx

```diff
@@ -133,7 +133,7 @@ export const CatalogSourceList = withFallback((props: CatalogSourceListProps) =>
 
   const data = [...props.data, ..._.get(props.globalCatalogSource, 'data', [])];
 
-  return props.loaded
+  return props.loaded || props.loadError
```

Comment:
I assume the load error here is for listing all config maps in the cluster. Can you help me understand why we need that?

If it's not required, should be make the resource `optional` when passing to `MultiListPage`?
If it is required, should we show an error here?

Reply:
Each `CatalogSource` has a corresponding `ConfigMap`. If we are viewing "All Namespaces", then we fetch all `CatalogSources` and all `ConfigMaps`; it has `namespaced: true` in the `Firehose` so it shouldn't be fetching all when viewing a single namespace.